lib/cc/service.rb

```diff
@@ -69,10 +69,11 @@ def self.slug
       end
     end
 
-    def initialize(config, payload)
-      @payload = payload.stringify_keys
-      @config  = create_config(config)
-      @event   = @payload["name"].to_s
+    def initialize(config, payload, beta_service_features)
```

Comment:
I agree with the name change. I don't think it needs to be optional, because the app only instantiates this object in one place)